corehq/apps/locations/fixtures.py

```diff
@@ -79,6 +84,17 @@ def __call__(self, restore_user, version, last_sync=None, app=None):
         if not should_sync_locations(last_sync, all_locations, restore_user):
             return []
 
+        xml_nodes = []
+        for serializer in self.serializers:
+            xml_nodes.extend(serializer.get_xml_nodes(restore_user, all_locations))
```

Comment:
yeah I was being a bit paranoid so this adds a fixture instead of replacing it. the two have different top-level IDs and should be completely isolated (it just looks like the one in the description below the normal one). it might make sense to just force you to use one or the other though.